src/openrct2/interface/Screenshot.cpp

```diff
@@ -25,6 +25,7 @@
 #include "../drawing/X8DrawingEngine.h"
 #include "../localisation/Formatter.h"
 #include "../localisation/Localisation.h"
+#include "../paint/Paint.h"
```

Comment:
This include is no longer needed, now with the `extern` lines below.

Reply:
There were other unnecessary includes so I might as well remove them too.